SECloseVoteRequestGenerator.user.js

```diff
@@ -1516,6 +1526,24 @@
             thisGuiItem.userChangedRequestType = true;
             thisGuiItem.adjustDisplayToRequestReason();
         });
+        this.SDReport.on('change', function() {
+            var reason = $('.cvrgReasonRow').find('input[type=text]').val();
+            if ($(this).is(':checked')) {
+                $('.cvrgReasonRow').find('input[type=text]').val(reason + ' (SD Report)')
```

Comment:
In this line and all lines below where you use `$('.cvrgReasonRow').find('input[type=text]')`, instead use the `requestReasonInput` variable created above. 

The current code really isn't going to do what you're intending. `$('.cvrgReasonRow').find('input[type=text]')` will find every request reason input on the page, which isn't desired, as it will result in changing the value of every request reason on the page.

IMO, it would be good if this code checks for "(SD Report)" already existing in the request reason and doesn't duplicate it if it's already there. It's debatable if it should also check for the single character substitution which produces "(SD Report)". Perhaps, it should check for "(SD Report)", or `/\bSD report\b/i` in the rendered preview and not add it if it's already in the preview.